README.md

```diff
@@ -0,0 +1,20 @@
+
+# Xmas text game short summary
+
+## Overview 
+This code is for python and uses if statements, functions, inputs, print statements, and time to make the project
+
+## Requiments
+Python 3 required 
+
+no modules required
+## Usage
+This is a text based game, so all you need to do is type in the
+
+number that is behind your answer.
+## Contribution
+Commit directly from the main branch
+
+Make a pull request about your branch
+
+This code is for python and uses if statements, functions, inputs, print statements, and time to make the project
```

Comment:
This is simply a duplication of your overview, why? I would add more about your PR instead, like add at least one reviewer. Also, you can add a note that is to say to search for open `issues` and refer to those in your PR